pgsql/pc_inout.c

```diff
@@ -313,6 +314,37 @@ Datum pcpatch_bytea_envelope(PG_FUNCTION_ARGS)
 	PG_RETURN_BYTEA_P(wkb);
 }
 
+PG_FUNCTION_INFO_V1(pcpatch_bytea_bounding_diagonal);
+Datum pcpatch_bytea_bounding_diagonal(PG_FUNCTION_ARGS)
+{
+	SERIALIZED_PATCH *serpatch;
+	PCSCHEMA *schema;
+	PCPATCH *patch;
+	uint8 *bytes;
+	size_t bytes_size;
+	bytea *wkb;
+	size_t wkb_size;
+
+	serpatch = PG_GETARG_SERPATCH_P(0);
```

Comment:
You should not deserialize the whole patch. See the stats access functions for deserializing only the stats.